crates/bevy_text/src/text.rs

```diff
@@ -127,7 +52,7 @@ impl Text {
 }
 
 /// Contains the value of the text in a section and how it should be styled.
```

Comment:
This is the first time we are adding a parent/child hierarchy requirement as far as I know, the docs should  indicate this on the related components.

Also with this change, the name "Text" is actually more confusing than helping, since it only contains the information for how text will be justified, and contains no text whatsoever